binstar_client/utils/conda.py

```diff
@@ -10,6 +10,22 @@
                  'conda.exe' if WINDOWS else 'conda')
 
 
+# this function is broken out for monkeypatch by unit tests,
+# so we can test the ImportError handling
+def _import_conda_root():
+    import conda.config
+    return conda.config.root_dir
+
+
+def _conda_root_from_conda_info():
+    try:
+        output = subprocess.check_output([CONDA_EXE, 'info', '--json']).decode("utf-8")
```

Comment:
The actual fix is this `decode("utf-8")`, I guess that was obscured by rearranging the code in order to add tests.